ts/Series/Contour/ContourSeries.ts

```diff
@@ -0,0 +1,915 @@
+     * - contourOffset,
+     * - smoothColoring,
+     * - showContourLines,
+     * - contourLineColor
+     * - colorAxisStops
+     * - colorAxisStopsCount
+     * - isInverted
+     */
+
+
+    /* *
+     *
+     * Methods
+     *
+     * */
+
+    public override init(chart: Chart, options: ContourSeriesOptions): void {
+        const props = {
+            minPadding: 0,
+            maxPadding: 0,
+            tickInterval: 1,
+            gridLineWidth: 0,
+            endOnTick: false,
+            startOnTick: false,
+            tickWidth: 1,
+            lineWidth: 1
+        };
+
+        merge(chart.yAxis[0].userOptions, props);
+        merge(chart.yAxis[0].options, props);
+
+        super.init(chart, options);
+
+
+        for (const axis of [this.xAxis, this.yAxis]) {
+            for (const [key, val] of Object.entries(props)) {
+                if (axis.userOptions[key as keyof typeof props] === void 0) {
+                    (axis.userOptions as any)[key] = (
+                        (axis.options as any)[key] = val
+                    );
+                }
+            }
+        }
+    }
+
+    public getContourData(): [Uint32Array, Float32Array] {
+        const points = this.points,
+            len = points.length,
+            points3d: Float32Array = new Float32Array(len * 3),
+            points2d: Float64Array = new Float64Array(len * 2),
+            { xAxis, yAxis } = this,
+            xDivider = (Math.abs(xAxis.toValue(0, true)) > 10e6) ?
+                10e6 :
+                1,
+            yDivider = (Math.abs(yAxis.toValue(yAxis.len, true)) > 10e6) ?
+                10e6 :
+                1;
+
+        for (let i = 0; i < len; i++) {
+            const { x, y = 0, value } = points[i],
+                index2d = i * 2,
+                index3d = i * 3;
+
+            points2d[index2d] = x / xDivider;
+            points2d[index2d + 1] = y && (y / yDivider) || 0;
+
+            points3d[index3d] = x;
+            points3d[index3d + 1] = y;
+            points3d[index3d + 2] = value ?? 0;
+        }
+
+        return [new Delaunay(points2d).triangles, points3d];
+    }
+
+    public override render(): void {
+        const { chart, options } = this,
+            visibility: 'hidden'|'inherit'|'visible' = this.visible ?
+                'inherit' : 'hidden',
+            zIndex = options.zIndex;
+
+        const targetGroup = chart.seriesGroup;
+
+        this.plotGroup(
+            'group',
+            'series',
+            visibility,
+            zIndex,
+            targetGroup
+        );
+
+        const { group } = this;
+        if (group && group?.parentGroup !== targetGroup) {
+            group.parentGroup = targetGroup;
+            // Can be improved by checking the exact zIndex of the series and
+            // placing it after the first element with a lower zIndex.
+            targetGroup?.element.prepend(group.element);
+        }
+
+        super.render();
+    }
```

Comment:
Not sure what this override does... When I remove it, the demos look the same. Is it a rudiment after an abandoned development step?